src/modules/fancyzones/dll/dllmain.cpp

```diff
@@ -143,6 +143,24 @@ class FancyZonesModule : public PowertoyModuleIface
             m_app->Destroy();
             m_app = nullptr;
             m_settings->ResetCallback();
+
+            if (s_llKeyboardHook)
+            {
+                bool success = UnhookWindowsHookEx(s_llKeyboardHook);

```

Comment:
Do we need to store return value here ?

Reply:
Here and at line 158 we can just do `if (function())`